### PR TITLE
Accuracy Checker: delete file logging

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

**Comment by eaidova** [comment]
@vladimir-dudnik, please merge ASAP. It breaks calibration in Windows package

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/logging.py [hunk 1]
```diff
@@ -72,12 +72,6 @@ def emit(self, record):
         }
     },
     'handlers': {
-        'file': {
-            'level': 'ERROR',
-            'class': 'logging.FileHandler',
-            'formatter': 'detailed',
-            'filename': _DEFAULT_LOG_FILE,
```
Comment (IRDonch) [inline tools/accuracy_checker/accuracy_checker/logging.py]: Maybe remove `_DEFAULT_LOG_FILE` itself too?

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/logging.py]: Removed, thanks

**Comment by IRDonch** [comment]
Jenkins: ok to test

**Pull request merged**